EnglishFilter/sections/foreign.txt

```diff
@@ -883,6 +883,9 @@ fofmyanmar.com##center > a[rel="noopener noreferrer"] > img
 ! https://github.com/AdguardTeam/AdguardFilters/issues/49041
 myptt.cc##p[style="padding:0;margin: 5px 0;color:#ff0000;"] > a[href][target="_blank"][style^="display:inline-block;"] > img
 myptt.cc#%#//scriptlet("set-constant", "adsbygoogle.loaded", "true")
+! https://github.com/AdguardTeam/AdguardFilters/issues/49080
+fashionwaltz.com###single_reading
+fashionwaltz.com##div[style*="margin-top:20px;"][style*="text-align:center;"]
```

Comment:
Change a bit second rule to this `fashionwaltz.com##.entry-content > div.copyright ~ div[style*="text-align:center;"]`